docs/pages/operations-guide.md

```diff
@@ -639,15 +680,15 @@ INFO  2017-04-25 20:26:08,343 [main] com.mesosphere.sdk.config.DefaultConfigurat
              "ranges" : null,
 ```
 
-The above steps apply to any configuration change: the Scheduler is restarted, detects the config change, and then launches and/or restarts any affected tasks to reflect the change. When multiple tasks are affected, the Scheduler will follow the deployment Plan used for those tasks to redeploy them. In practice this typically means that each task will be deployed in a sequential rollout, where task `N+1` is only redeployed after task `N` appears to be healthy after being relaunched with the new configuration.
+The steps above apply to any configuration change: the Scheduler is restarted, detects the config change, and then launches and/or restarts any affected tasks to reflect the change. When multiple tasks are affected, the Scheduler will follow the deployment Plan used for those tasks to redeploy them. In practice this typically means that each task will be deployed in a sequential rollout, where task `N+1` is only redeployed after task `N` appears to be healthy after being relaunched with the new configuration.
 
 ### Add a node
 
-Adding a task node to the service is just another type of configuration change. In this case we're looking for a specific config value in the package's `config.json`, and then mapping that configuration value to the relevant environment variable in the Scheduler. In the case of the above `dse` service, it'd just be a matter of increasing the Scheduler's `DSE_NODE_POD_COUNT` from `3` (the default) to `4`. After the change, the Scheduler will deploy a new DSE node instance without changing the preexisting nodes.
+Adding a task node to the service is just another type of configuration change. In this case, we're looking for a specific config value in the package's `config.json`, and then mapping that configuration value to the relevant environment variable in the Scheduler. In the case of the above `dse` service, we need to increase the Scheduler's `DSE_NODE_POD_COUNT` from `3` (the default) to `4`. After the change, the Scheduler will deploy a new DSE node instance without changing the preexisting nodes.
 
 ### Finding the correct environment variable
 
-The correct environment variable for a given setting can vary depending on the service. For example with increasing the number of nodes, some services will have multiple types of nodes, each with separate count settings. Determining the correct environment variable in this case takes some detective work.
+The correct environment variable for a given setting can vary depending on the service. For example, with increasing the number of nodes, some services will have multiple types of nodes, each with separate count settings. Determining the correct environment variable in this case takes some detective work.
```

Comment:
The second sentence now has three commas -- I can't think of a better way to structure it though...